modules/EmailAddresses/EmailAddress.php

```diff
@@ -52,20 +52,56 @@
  */
 class EmailAddress extends SugarEmailAddress
 {
-	var $disable_row_level_security = true;
+    /**
+     * @var bool $disable_row_level_security
+     */
+	public $disable_row_level_security = true;
 
-	function __construct()
+    /**
+     * @var int|bool $opt_out
+     */
+    public $opt_out = 0;
+
+    /**
+     * @var int|bool $confirm_opt_in
+     */
+	public $confirm_opt_in = 0;
+
+	/**
+     * @var int|bool $invalid_email
+     */
+	public $invalid_email = 0;
+
+	/**
+     * @var TimeDate $opt_in_email_created
+     */
+    public $opt_in_email_created;
```

Comment:
Where does it get any value?

Reply:
Please explain?

opt_in_email_created is defined in the var_defs or rather in the metadata/email_addressesMetaData.php file. So I have added the property to make it easier to find with and IDE.

Reply:
Thats ok, my problem is I don't find anywhere in code base or in this PR where we using this variable. Where do you set the value of it?